Library/BookCopy.aspx.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Library
+{
+    public partial class BookCopy : System.Web.UI.Page
+    {
+        int bookID = 0;
+        int library_id = 0;
+        
+
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            if (!int.TryParse(Request.QueryString["ID"], out bookID))
+            {
+                Response.Redirect("~/BookList.aspx");
+            }
+
+            if (!IsPostBack)
+            {
+                DataTable dt = DatabaseHelper.Retrieve(@"
+                    select Book.Title, Book.ISBN, Author.FirstName + ' ' + Author.LastName as Name
+                    from Book
+                    inner join Author
+                        on Author.ID = Book.Author_ID
+                    where Book.ID = @ID
+                ", new SqlParameter("@ID", bookID));
+
+                if (dt.Rows.Count == 1)
+                {
+                    BookTitle.Text = dt.Rows[0].Field<string>("Title");
+                    ISBN.Text = dt.Rows[0].Field<string>("ISBN");
+                    Author.Text = dt.Rows[0].Field<string>("Name");
+                    BookTitle.Enabled = false;
+                    ISBN.Enabled = false;
+                    Author.Enabled = false;
+                }
+                else
+                {
+                    Response.Redirect("~/BookList.aspx");
+                }
+
+                DataTable dt2 = DatabaseHelper.Retrieve(@"
```

Comment:
When you have two or more data tables in scope within a method, I'd use variable names that are more descriptive than `dt` and `dt2`.